content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -38,8 +38,16 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
 
     ```
     export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/crd.yaml
+    kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+    kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml
```

Comment:
IMO
```
kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
```

It's easier to find source code with these instructions.

What do you think?

Reply:
Good idea. Thanks!